submissions/semenjob/script/index.js

```diff
@@ -0,0 +1,52 @@
+window.addEventListener('DOMContentLoaded', () => {
+  initMusicApp()
+})
+
+function initMusicApp() {
+  const tabs = document.querySelectorAll('.wrapper__item'), contentTitle = document.querySelector('.music__title'), contentDescr = document.querySelector('.music__descr'), parentItems = document.querySelector('.wrapper__list')
+
```

Comment:
The load function should only have initial function calls, description of some variables, possibly initial checks

Reply:
but here, to be honest, I didn’t understand how to solve this issue, is there any kind of thing?